airflow/providers/microsoft/azure/hooks/container_instance.py

```diff
@@ -177,6 +177,8 @@ def get_logs(self, resource_group: str, name: str, tail: int = 1000) -> list:
         :return: A list of log messages
         """
         logs = self.connection.containers.list_logs(resource_group, name, name, tail=tail)
+        if logs.content is None:
```

Comment:
@krisfur Wouldn't this still raise an `AttributeError`? Maybe we can do something like
```suggestion
        if not getattr(logs, "content", None):
```